src/components/generic/movieModal/modalContent.tsx

```diff
@@ -97,18 +101,18 @@ const ModalContent: FC<Props> = ({
               </Category>
             </Modal.Center>
             <Modal.Right>
-              <Category title="Trailer">
-                <iframe
+              {/* <Category title="Trailer"> */}
+              {/* <iframe
                   title="trailer"
                   width="100%"
                   height="250"
                   src={`https://www.youtube.com/embed/${trailer}`}
                   frameBorder="0"
                   allowFullScreen
                   allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-                />
-              </Category>
-              <SelectQuality links={items} imdb={imdb} />
+                /> */}
+              {/* </Category> */}
+              {/* <SelectQuality links={items} imdb={imdb} /> */}
```

Comment:
I suggest not leaving huge comments like these UNLESS you leave a comment like 

```
// TODO: Explain blah blah blah
```

or 

```
// NOTE: Explain blah blah blah
```

`TODO`s can be detected by modern code editors.

Reply:
Done